src/protocols/aeternity/composables/aeTippingUrls.ts

```diff
@@ -11,7 +11,7 @@ function getTwitterAccountUrl(url: string) {
 const verifiedUrls = ref<string[]>([]);
 const blacklistedUrls = ref<string[]>([]);
 
-export function useAeTippingUrls() {
+export function useAeTippingUrls(ensureFetchedOnInit = true) {
```

Comment:
It's more common to pass the composable options as an object to make it easy to extend and modify it.